CHANGES_NEXT_RELEASE

```diff
@@ -31,6 +31,7 @@
 - Fix: Better error returned on attribute not found in GET /v2/entities/<entity-id>/attrs/<attr-name>/value (Issue #2220)
 - Fix: Distinguish between strings and numbers in q string filter (Issue #1129)
 - Fix: proper response for 'GET /v2/entities/<id>?attrs=<attrName>' and 'GET /v2/entities/<id>/attrs?attrs=<attrName>' when <attrName> doesn't exist (#2241)
+- Fix: throttling set to zero (Issue #2030) (This issue was actually fixed for the release 1.2.0)
```

Comment:
Given this is a change in an already release version, this entry should be moved to the 1.2.0 section in contextBroker.spec.